### ISSUE TITLE
User can enter the app without actually logging in

### ISSUE THREAD

**Issue by darjalsk** [letsdoitworld/World-Cleanup-Day#278]
**Device:** iPhone 8+ 11.3.1
**Pre-conditions:** Release build, branch hotfix/blue_screen

**Steps to reproduce:**
Select to log in with Facebook 
Turn off the internet connection -> "Welcome to offline mode" message is shown
Close the message - person is in the app

**Actual result:** User can enter the app without actually logging in
**Expected result:** User cannot enter the app without logging in first.

**Comment by lynxlynxlynx** [comment]
Why is that unexpected? The main screen has an explicit guest-mode option.

**Comment by darjalsk** [comment]
It is unexpected, because without logging in you get to view all the functionality available. You can even try to create trashpoint. Eventually, user is kicked out to the login screen, but it is not Ok, in my opinion, because in guest mode, we don’t have an ability to view this functionality.

**Comment by KristiinaKerge** [comment]
True, this is not expected behavior but we can release with it as we don't have much time

**Comment by KristiinaKerge** [comment]
is it solved in new version
